runtime/curl.cpp

```diff
@@ -667,8 +670,11 @@ mixed f$curl_exec(curl_easy easy_id) noexcept {
   easy_context->error_num = dl::critical_section_call(curl_easy_perform, easy_context->easy_handle);
   double request_finish_time = dl_time();
   if (request_finish_time - request_start_time >= long_curl_query) {
-    kprintf("LONG curl query : %f. Curl id = %d, url = %.100s\n", request_finish_time - request_start_time, easy_context->uniq_id,
-            easy_context->get_info(CURLINFO_EFFECTIVE_URL).as_string().c_str());
+    string curl_url = easy_context->get_info(CURLINFO_EFFECTIVE_URL).as_string();
+    StatsHouseManager::get().add_slow_net_event_stats(
+        slow_net_event_stats::slow_curl_response_stats{slow_net_event_stats::slow_curl_response_stats::curl_kind::sync,
+                                                       std::string_view{curl_url.c_str(), curl_url.size()}, request_finish_time - request_start_time});
```

Comment:
I think we need to normalize url here somehow to get rid of very long urls. Maybe let's cut it up to the first 100 symbols?